<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignalInfoDlg</class>
 <widget class="QDialog" name="SignalInfoDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>235</width>
    <height>222</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Signal Info</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QLabel" name="lbRangeMax">
     <property name="text">
      <string>Max Range</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QDoubleSpinBox" name="spinRangeMax">
     <property name="maximum">
      <double>99999999999.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lbTicks">
     <property name="text">
      <string>Ticks</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QDoubleSpinBox" name="spinRangeMin">
     <property name="decimals">
      <number>2</number>
     </property>
     <property name="minimum">
      <double>-99999999999.000000000000000</double>
     </property>
     <property name="maximum">
      <double>99999999999.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.500000000000000</double>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="lbRangeMin">
     <property name="text">
      <string>Min Range</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="editTicks"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lbTickLabels">
     <property name="text">
      <string>Tick Labels</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="editTickLabels"/>
   </item>
   <item row="4" column="0" colspan="2" alignment="Qt::AlignHCenter">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SignalInfoDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SignalInfoDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
